docs/test/live-unit-testing-start.md

```diff
@@ -352,7 +353,7 @@ In this section, you'll explore how you can use Live Unit Testing to identify, t
 
 ---
 
-This provides enough information for a preliminary investigation of the bug. Either `TestHasEmbeddedSpaces`, the test routine, made an incorrect assumption, or `HasEmbeddedSpaces` does not correctly recognize all embedded spaces. To diagnose and correct the problem, start with the `StringLibrary.HasEmbeddedSpaces` method:
+This provides enough information for a preliminary investigation of the bug. Either `TestHasEmbeddedSpaces` (the test routine), made an incorrect assumption, or `HasEmbeddedSpaces` does not correctly recognize all embedded spaces. To diagnose and correct the problem, start with the `StringLibrary.HasEmbeddedSpaces` method:
```

Comment:
the comma after "(the test routine)" should be deleted.

Reply:
I've pushed this change.